Doc/whatsnew/3.8.rst

```diff
@@ -593,20 +594,22 @@ builtins
 
 The :func:`compile` built-in has been improved to accept the
 ``ast.PyCF_ALLOW_TOP_LEVEL_AWAIT`` flag. With this new flag passed,
-:func:`compile` will allow top-level ``await``, ``async for`` and ``async with``
-constructs that are usually considered invalid syntax. Asynchronous code object
-marked with the ``CO_COROUTINE`` flag may then be returned.
+:func:`compile` will allow top-level :keyword:`await`, :keyword:`async for` and
+:keyword:`async with` constructs that are usually considered invalid syntax.
+Asynchronous code object marked with the :data:`inspect.CO_COROUTINE` flag may
```

Comment:
This doesn't read well, probably insert "An" as the first word of the sentence.

Reply:
I agree that the current version doesn't read well, but I would prefer changing "object" to "objects":
```
Asynchronous code objects marked with the :data:`inspect.CO_COROUTINE` flag may
```

Both options would be grammatically correct, it's fairly subjective. The main reason I'm leaning towards the plural here is because there can be multiple asynchronous code objects marked within the code being compiled, not just one.

Reply:
Yes, that's better